Makefile.PL

```diff
@@ -40,7 +40,7 @@ my %WriteMakefileArgs = (
     "Test::More" => "0.94",
     "Test::Warnings" => 0
   },
-  "VERSION" => "0.06",
+  "VERSION" => "0.08",
```

Comment:
Will any service be affected by upgrading this version?

Reply:
@afshin-deriv I dont think so, the changes are minimal and only concern a specific type of response which is not being used in our production yet.

Reply:
Curiosity question: the `FIX44.xml` was originally taken from some other project, so, it is kinda standard. What you are trying to do now, is change standard at you own will, may be with external requirements. I'd suggest to add link to some public document, where it is stated that this field have to be presented.

If the change is caused only by external requirements, I'd suggest to check the developed extension mechanism in the library https://github.com/binary-com/perl-Protocol-FIX#extensionself-extension_path , to check, whether it is ok or not for you.

PS. Hello to guys in Cyberjaya. )

Reply:
Hi @basiliscos thanks for your comment.
The schema lacks the Instrument field for the `MarketDataIncrementalRefresh` completely , even though its not always a  required field, an standard fix engine needs to at least have the option to provide an "Instrument" in the  `MarketDataIncrementalRefresh` response message, please refer to the structure mentioned [here](https://www.onixs.biz/fix-dictionary/4.4/msgType_X_88.html).
regarding the usage of extension mechanism, I believe it would be a bit overkill since we dont have any custom response type or schema in our engine. and the changes introduced here can be considered as an improvement to this protocol code, in the current version without this improvement  one simply can not generate a `MarketDataIncrementalRefresh` message for a specific symbol unless they define their own schema for all their messages.